src/components/layouts/AppLayout.vue

```diff
@@ -1,10 +1,242 @@
+              >
+                Crafted by HitPay
+                <a
+                  href="https://hitpayapp.com"
+                  class="font-semibold text-indigo-600"
+                  ><span class="absolute inset-0" aria-hidden="true" />Read more
+                  <span aria-hidden="true">&rarr;</span></a
+                >
+              </div>
+            </div>
+            <p class="mt-6 text-lg leading-8 text-gray-600">
+              The Open Source UI Library for Vue3 offers a wide range of
+              components and tools to streamline the development process for
+              Vue.js applications. With a focus on flexibility and
+              customization, this library provides developers with the ability
+              to create modern and responsive user interfaces with ease.
+              <br />
+              <br />
+              Whether you are building a simple website or a complex web
+              application, this UI library has everything you need to enhance
+              the user experience and improve overall functionality.
+            </p>
+            <div class="mt-10 flex items-center justify-center gap-x-6">
+              <a
+                href="https://github.com/hit-pay/orchid"
+                class="rounded-md bg-slate-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-slate-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-slate-600"
+                >GitHub ⭐
+              </a>
+              <a
+                href="/storybook/"
+                class="rounded-md bg-indigo-600 px-3.5 py-2.5 text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
+                >StoryBook ⭐
+              </a>
+            </div>
+          </div>
+        </div>
+        <div
+          class="absolute inset-x-0 top-[calc(100%-13rem)] -z-10 transform-gpu overflow-hidden blur-3xl sm:top-[calc(100%-30rem)]"
+          aria-hidden="true"
+        >
+          <div
+            class="relative left-[calc(50%+3rem)] aspect-[1155/678] w-[36.125rem] -translate-x-1/2 bg-gradient-to-tr from-[#ff80b5] to-[#9089fc] opacity-30 sm:left-[calc(50%+36rem)] sm:w-[72.1875rem]"
+            style="
+              clip-path: polygon(
+                74.1% 44.1%,
+                100% 61.6%,
+                97.5% 26.9%,
+                85.5% 0.1%,
+                80.7% 2%,
+                72.5% 32.5%,
+                60.2% 62.4%,
+                52.4% 68.1%,
+                47.5% 58.3%,
+                45.2% 34.5%,
+                27.5% 76.7%,
+                0.1% 64.9%,
+                17.9% 100%,
+                27.6% 76.8%,
+                76.1% 97.7%,
+                74.1% 44.1%
+              );
```

Comment:
If the polygon is same as above in code could we maybe create variable to it to make html more clear, WDYT @kewcoder?

Reply:
This is random ui, I copied it from tailwindui component https://tailwindui.com/components/marketing/sections/heroes, maybe later we can improve it again.